tokenserver/tests/test_service.py

```diff
@@ -88,6 +117,9 @@ def _getassertion(self, **kw):
         kw.setdefault('audience', 'http://tokenserver.services.mozilla.com')
         return make_assertion(**kw).encode('ascii')
 
+    def _gettoken(self, email='tarek@mozilla.com'):
```

Comment:
maybe "example@mozilla.com" instead? tarek probably gets enough email as is.

Reply:
lol, the tests have a bunch of existing references to tarek but yeah, it's probably time for them to go :-)